core/server/api/canary/authentication.js

```diff
@@ -190,5 +197,14 @@ module.exports = {
                     return models.Invite.findOne({email: email, status: 'sent'}, frame.options);
                 });
         }
+    },
+
+    resetAllPasswords: {
+        // validation: false,
```

Comment:
@tpatel what's the reason behind this being commented out?

Reply:
It's definitely a leftover from early attempts at learning the Ghost framework, I've removed this because uncommenting would skip the validation step and there is no data to validate!